<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1270</width>
    <height>905</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1270</width>
    <height>905</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1270</width>
    <height>905</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Classroom Management</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCalendarWidget" name="calendar">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>60</y>
      <width>321</width>
      <height>191</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>FiraCode Nerd Font Mono Med</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(33, 75, 57);
color:white;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>421</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>FiraCode Nerd Font Mono Ret</family>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Classroom Management</string>
    </property>
   </widget>
   <widget class="QPushButton" name="teacher_btn">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>70</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>FiraCode Nerd Font Mono Ret</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(33, 75, 57);
color:white;
border-style:outset;
border-radius:9px;
border-width:2px;
border-color:black;</string>
    </property>
    <property name="text">
     <string>Teacher</string>
    </property>
   </widget>
   <widget class="QPushButton" name="room_btn">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>130</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>FiraCode Nerd Font Mono Ret</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(33, 75, 57);
color:white;
border-style:outset;
border-radius:9px;
border-width:2px;
border-color:black;</string>
    </property>
    <property name="text">
     <string>Room</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="time_table_dashboard">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>1241</width>
      <height>591</height>
     </rect>
    </property>
    <attribute name="verticalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QPushButton" name="booking_btn">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>190</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>FiraCode Nerd Font Mono Ret</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(33, 75, 57);
color:white;
border-style:outset;
border-radius:9px;
border-width:2px;
border-color:black;
</string>
    </property>
    <property name="text">
     <string>Booking</string>
    </property>
   </widget>
   <widget class="QPushButton" name="refresh_btn">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>223</y>
      <width>75</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>FiraCode Nerd Font Mono Ret</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(33, 75, 57);
color:white;
border-style:outset;
border-radius:9px;
border-width:2px;
border-color:black;
</string>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1270</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
